Mathlib/LinearAlgebra/CrossProduct.lean

```diff
@@ -56,55 +56,70 @@ def crossProduct : (Fin 3 → R) →ₗ[R] (Fin 3 → R) →ₗ[R] Fin 3 → R :
   · intros
     simp_rw [smul_vec3, Pi.smul_apply, smul_sub, mul_smul_comm]
 
-@[inherit_doc] scoped[Matrix] infixl:74 " ×₃ " => crossProduct
+@[inherit_doc] scoped[Matrix] infixl:74 " ⨯ " => crossProduct
+
+namespace Matrix
+/-- A deprecated notation for `⨯`. -/
+@[deprecated «term_⨯_» (since := "2025-07-11")]
+scoped syntax:74 (name := _root_.«term_×₃_») term:74 " ×₃ " term:75 : term
+end Matrix
+
+open Lean Elab Meta.Tactic Term in
+@[term_elab Matrix._root_.«term_×₃_», inherit_doc «term_×₃_»]
+def elabDeprecatedCross : TermElab
+| `($x ×₃%$tk $y) => fun ty? => do
+  logWarningAt tk <| .tagged ``Linter.deprecatedAttr <| m!"The ×₃ notation has been deprecated"
+  TryThis.addSuggestion tk { suggestion := "⨯" }
+  elabTerm (← `($x ⨯ $y)) ty?
```

Comment:
I was reminded of `Lean.Elab.Term.adaptExpander`.

I think the way this looks is something like
```lean
def elabDeprecatedCross : TermElab := adaptExpander fun
  | `($x ×₃%$tk $y) => do
    logWarningAt tk <| .tagged ``Linter.deprecatedAttr "The ×₃ notation has been deprecated"
    TryThis.addSuggestion tk { suggestion := "⨯" }
    `($x ⨯ $y)
  | _ => throwUnsupportedSyntax
```